ports/esp32/mpconfigport.h

```diff
@@ -306,7 +310,7 @@ void boardctrl_startup(void);
 
 #if MICROPY_PY_NETWORK_LAN && CONFIG_ETH_USE_SPI_ETHERNET
 #ifndef MICROPY_PY_NETWORK_LAN_SPI_CLOCK_SPEED_MZ
-#if CONFIG_IDF_TARGET_ESP32 || CONFIG_IDF_TARGET_ESP32C3 || CONFIG_IDF_TARGET_ESP32C2
+#if CONFIG_IDF_TARGET_ESP32 || CONFIG_IDF_TARGET_ESP32C3 || CONFIG_IDF_TARGET_ESP32C2 || CONFIG_IDF_TARGET_ESP32C6
```

Comment:
I can't figure out what this is selecting for. The division has been here in MicroPython since support was added in #10420, but MicroPython always picks the default clock source is used (SPI_CLK_SRC_DEFAULT) and this is APB clock on all these SoCs (40MHz)

The ESP-IDF examples default to 36MHz for all chips:
https://github.com/espressif/esp-idf/blob/master/examples/common_components/protocol_examples_common/Kconfig.projbuild#L283

@damz I don't suppose you remember why you had to use this conditional logic?

Reply:
@damz Thanks for that, much appreciate! Defaulting to 20MHz seems reasonable to me, too.